common/Cargo.toml

```diff
@@ -107,6 +107,7 @@ base64 = { workspace = true }
 hyper = { workspace = true }
 proptest = "1.1.0"
 ring = { workspace = true }
+shuttle-common-tests = { workspace = true }
```

Comment:
I'm not sure if we would rather want to inverse this dependency :thinking:

Reply:
I'm worried we want access to `shuttle-common` in the tests :sweat_smile: I guess we can't just move things around if / when that is a problem